### PR TITLE
fix legend bugs

### PR DESCRIPTION
**Pull request by kbspooner**

### CODE REVIEW

**Comment by dandavies99** [comment]
Sorry @kbspooner - are you still waiting for a review from me or is this PR old news now?

**Comment by kbspooner** [comment]
Oh I think I must have done this as a formality... mosly just typos and stuff... if Travis says yes its probably good to pull!

**Review by dandavies99 — APPROVED** [review]

**Pull request merged**
